yarn.lock

```diff
@@ -4637,6 +4654,16 @@ __metadata:
   languageName: node
   linkType: hard
 
+"changeset@npm:^0.2.6":
```

Comment:
Don't think any of these changes should happen in `yarn.lock` due to introduction of JSON Schema.